formats/cbor/commonMain/src/kotlinx/serialization/cbor/CborDecoder.kt

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright 2017-2024 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.serialization.cbor
+
+public interface CborDecoder {
+    @OptIn(kotlinx.serialization.ExperimentalSerializationApi::class)
```

Comment:
Since this interface will be experimental too, we should mark it with this annotation instead of opting into it.

Reply:
my bad. fixed in 176848cfd870a7fd98f40743a5998844f8772933